client/flutter/lib/api/content/schema/index_content.dart

```diff
@@ -55,17 +66,56 @@ class IndexPromo {
     @required this.subtitle,
     @required this.link,
     @required this.buttonText,
+    this.promoType,
   });
+
+  IndexPromoType get type {
+    switch (this.promoType) {
+      case 'check_your_symptoms':
+        return IndexPromoType.CheckYourSymptoms;
+        break;
+      case 'protect_yourself':
+        return IndexPromoType.ProtectYourself;
+        break;
+    }
+    return IndexPromoType.DefaultType;
+  }
+}
+
+enum IndexItemType {
+  recent_numbers,
+  protect_yourself,
+  information_card,
+  unknown
 }
 
 class IndexItem {
+  final String itemType;
   final String title;
   final String subtitle;
   final RouteLink link;
+  final String buttonText;
 
   IndexItem({
-    @required this.title,
-    @required this.subtitle,
-    @required this.link,
+    this.itemType,
+    this.title,
+    this.subtitle,
+    this.link,
+    this.buttonText,
   });
+
+  IndexItemType get type {
+    switch (this.itemType) {
+      case 'recent_numbers':
+        return IndexItemType.recent_numbers;
+        break;
```

Comment:
Is this a Dart thing?  Break after return?

Reply:
I'd thought that I'd read break was required even with return statements in Dart, but I must have been mistaken... removed